contracts/Insurance.sol

```diff
@@ -200,18 +200,13 @@ contract Insurance is IInsurance {
     /**
      * @notice Gets the 8 hour funding rate for an insurance pool
      * @dev the funding rate is represented as
-     *      0.0036523 * (insurance_fund_target - insurance_fund_holdings) / leveraged_notional_value)
+     *      (182.648 / 8) * (5 * ((fundTarget - fundHoldings) / (fundTarget * 10_000))) ** 2
```

Comment:
This could throw an error. div by 0/ Fund holdings could be positive

Reply:
It could by this alone, but below there is a check for
```
        if (poolTarget <= poolHoldings) {
            return 0;
        }
```
if `poolTarget` is 0, then poolHoldings must be at least == pool target, and therefore will enter this branch and return 0